jdaviz/configs/default/plugins/viewers.py

```diff
@@ -166,6 +166,17 @@ def set_limits(self, x_min=None, x_max=None, y_min=None, y_max=None):
             if y_max is not None:
                 self.state.y_max = y_max
 
+    def get_limits(self):
```

Comment:
I find this very convenient, so might as well add it in this PR. Otherwise, I will never get to it.